src/utils/uniq.ts

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Can we move this comment to where we're defining `uniq` so it shows up in intellisense?